src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1109,6 +1159,8 @@ private static String addPrefix(String server) {
      * See http://msdn.microsoft.com/en-us/library/aa746475(VS.85).aspx for the syntax by example.
      * WANTED: The specification of the syntax.
      */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", 
+            justification = "Filter may be changed in the runtime by scripts if required")
```

Comment:
Anyway, it should be done by a separate pull request, because it is not a "code" polishing. @stephenc , do you agree with merging this PR as is? If yes, I'll create an issue for this field